src/formulas/tokenizer.ts

```diff
@@ -110,7 +108,15 @@ function tokenizeOperator(chars: TokenizingChars): Token | null {
   return null;
 }
 
+const FIRST_POSSIBLE_NUMBER_CHARS = new Set("0123456789");
```

Comment:
are the negative numbers tokenized as operator "-" & number (2 tokens) or should they be 1 token of negative number ?

Reply:
They are two tokens ("-" operator and positive number)